internal/driver/commands.go

```diff
@@ -196,10 +196,14 @@ var pprofVariables = variables{
 		"Matching includes the function name, filename or object name.")},
 	"tagfocus": &variable{stringKind, "", "", helpText(
 		"Restrict to samples with tags in range or matched by regexp",
-		"Discard samples that do not include a node with a tag matching this regexp.")},
+		"Discard samples that do not include a node with a tag matching this regexp.",
+		"Can specify both the name and value of the tag which should be matched",
```

Comment:
This doesn't mention numeric tag filtering and ability to use ranges.  Agree that the text would get too long if we just try to describe that. One thing we can do is just put examples of possible usage here.